### DISCUSSION TITLE
Does revalidatePath work as expected for you?

### DISCUSSION THREAD

**Discussion by codinginflow** [vercel/next.js#50781] [App Router]
No matter what string I pass to revalidatePath, it always seems to refresh the current page. It doesn't refresh any other routes and it doesn't seem to matter at all what string I pass to the function.

I know that I have to pass the folder structure, not the URL, e.g.:

```
revalidatePath("/produts/[id]");
```

but it is not working.

**Comment by BrendonSled** [comment]
I'm seeing a similar behaviour too. Trying to revalidate another path doesn't seem to work.

**Reply by BrendonSled** [reply]
Actually it looks like there's discussion about this here:
https://github.com/vercel/next.js/issues/49450#issuecomment-1546477184

**Reply by codinginflow** [reply]
Thank you for the link!
